pages/send/Manual.tsx

```diff
@@ -34,11 +53,95 @@ export function Manual() {
+                .map((addressBookEntry, index) => (
+                  <Pressable
+                    key={index}
+                    onPress={() => {
+                      router.dismissAll();
+                      router.navigate({
+                        pathname: "/send",
+                        params: {
+                          url: addressBookEntry.lightningAddress,
+                        },
+                      });
+                    }}
+                    className="mb-4"
+                  >
+                    <Card className="border-0 shadow-none">
+                      <CardContent className="flex flex-row items-center gap-4">
+                        <View className="h-10 w-10 flex items-center justify-center rounded-full bg-accent">
+                          <Text className="text-foreground text-base font-bold">
+                            {addressBookEntry.name?.[0]?.toUpperCase() ||
+                              addressBookEntry.lightningAddress?.[0]?.toUpperCase() ||
+                              "SN"}
+                          </Text>
+                        </View>
+                        <View className="flex flex-1 flex-col">
+                          <CardTitle>
+                            {addressBookEntry.name ||
+                              addressBookEntry.lightningAddress}
+                          </CardTitle>
+                          <CardDescription>
+                            {addressBookEntry.lightningAddress}
+                          </CardDescription>
+                        </View>
+                      </CardContent>
+                    </Card>
+                  </Pressable>
+                ))}
+            </ScrollView>
+          )}
+          {keyboardText.includes("@") &&
+            !addressBookEntries.some(
+              (entry) => entry.lightningAddress === keyboardText,
+            ) && (
+              <View className="flex items-center justify-center">
+                <Text className="text-muted-foreground text-center flex flex-row gap-4">
+                  <Checkbox
+                    checked={addToContacts}
+                    onCheckedChange={setAddToContacts}
+                  />{" "}
+                  <View>
+                    <Text>Add to contacts</Text>
+                  </View>
+                </Text>
+              </View>
+            )}
+          {addToContacts && keyboardText.includes("@") && (
+            <View>
+              <Label nativeID="name" className="self-start justify-self-start">
+                Name
+              </Label>
+              <Input
+                autoFocus
+                placeholder="Satoshi"
+                inputMode="email"
+                className="w-full"
+                value={contactName}
+                onChangeText={setContactName}
+                aria-labelledbyledBy="name"
+                returnKeyType="done"
+                // aria-errormessage="inputError"
+              />
```

Comment:
I think we should improve our input box styling to be in line with the rest of the app (because here we are merging it with the design instead of a separate screen)